core/arch/arm/kernel/boot.c

```diff
@@ -952,6 +952,7 @@ void init_tee_runtime(void)
 
 static void init_primary(unsigned long pageable_part, unsigned long nsec_entry)
 {
+	thread_init_thread_core_local();
```

Comment:
This function is already called in `_start()` (`core/arch/arm/kernel/entry_a64.S`) or `reset_primary()` (`core/arch/arm/kernel/entry_a32.S`). Do you need to call it again here?

Reply:
I'm sorry, I copied a bit much from another patch I had in the pipe. I'll remove it.

Reply:
You're right, fixed now.